packages/core/src/html/MdAttributeRenderer.ts

```diff
@@ -24,24 +29,28 @@ class MdAttributeRenderer {
    * @param isInline Whether to process the attribute with only inline markdown-it rules
    * @param slotName Name attribute of the <slot> element to insert, which defaults to the attribute name
    */
-  processAttributeWithoutOverride(node, attribute, isInline, slotName = attribute) {
+  processAttributeWithoutOverride(node: DomElement, attribute: string,
+                                  isInline: boolean, slotName = attribute): void {
     const hasAttributeSlot = node.children
         && node.children.some(child => getVslotShorthandName(child) === slotName);
 
-    if (!hasAttributeSlot && _.has(node.attribs, attribute)) {
+    const nodeAttribs = node.attribs ?? {};
```

Comment:
This is because `node.attribs` is potentially `undefined`.

Reply:
Should we also do the early return here when there are no attributes in the node? 
e.g. `if !(node.attribs && _.has(node.attribs, attribute) return`
Same for `hasSlotOverridingAttribute`

Reply:
@raysonkoh forgot to tag you on this

Reply:
Sorry, missed this 😅 . Fixed.